### DISCUSSION TITLE
Installation with conda?

### DISCUSSION THREAD

**Discussion by ananthsub** [Lightning-AI/pytorch-lightning#8990] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule] [answered]
I got this question from someone around installing Lightning with conda

Running both `conda install -c conda-forge pytorch-lightning`  and `conda update pytorch-lightning` resulted in the version 0.8.5 being used, instead of the latest (as of now, 1.4.2). https://anaconda.org/conda-forge/pytorch-lightning
I'm curious if anyone else has seen this issue. @Borda ?

**Comment by Borda** [comment, marked as answer]
Conda-forge does not have any latest pytorch (the last is 1.1) which in leads installing pytorch-lightning 0.8 as it was last supporting this PT version... You need to add channel pytorch
`conda install -c pytorch -c conda-forge pytorch-lightning`
